packages/ckeditor5-core/src/editor/editor.ts

```diff
@@ -1010,13 +1018,25 @@ export default abstract class Editor extends /* #__PURE__ */ ObservableMixin() {
 				/**
 				 * Your license does not allow the current distribution channel.
 				 *
+				 * These are the available distribution channels:
+				 * * Self-hosted - the editor is installed via npm or from a ZIP package
+				 * * Cloud - the editor is run from CDN
+				 *
+				 * The licenses available include:
+				 * * GPL license for open-source users
+				 * * Cloud plans (managed via the Customer Portal)
```

Comment:
Custom plans are also managed by the Customer Portal. Maybe just one group: `Commercial plans (Cloud or sales-assisted)`.